.github/workflows/continuous-integration.yml

```
@@ -19,9 +19,9 @@ jobs:
         python-version: [ 3.9, '3.10', '3.11', '3.12' ]
         isMerge:
           - ${{ github.event_name == 'push' && github.ref == 'refs/heads/devel' }}
-        exclude:
-          - { isMerge: false, python-version: '3.10' }
-          - { isMerge: false, python-version: '3.11' }
+        #exclude:
+        #  - { isMerge: false, python-version: '3.10' }
+        #  - { isMerge: false, python-version: '3.11' }
```

This was just a temporary test I made. It can be reverted now

```suggestion
        exclude:
          - { isMerge: false, python-version: '3.10' }
          - { isMerge: false, python-version: '3.11' }
```